ci/test/resources/build.yaml

```diff
@@ -149,7 +145,7 @@ steps:
    secrets:
     - name: test-tokens
       namespace:
-        valueFrom: batch_pods_ns.name
+        valueFrom: default_ns.name
```

Comment:
Where does this test-tokens secret come from? I guess this test depends on create_accounts because test_ci depends on that.

Reply:
This is the test build.yaml.  test-tokens is created by the outer build.yaml.